src/util/event-emitter.js

```diff
@@ -1,32 +1,38 @@
 /* global setTimeout */
 // # EventEmitter class
-// Simple re-implementation of Angular 2's [EventEmitter](https://github.com/angular/angular/blob/master/modules/angular2/src/core/facade/async.ts#L137)
-import Subject from '@reactivex/rxjs/dist/cjs/Subject';
+// Simple re-implementation of Angular 2's [EventEmitter](https://github.com/angular/angular/blob/master/modules/angular2/src/facade/async.ts#L97)
+import eventDispatcher from "./event-dispatcher.js";
+
+const uuid = function() {
+  return 'xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx'.replace(/[xy]/g, function(c) {
+    var r = Math.random()*16|0, v = c == 'x' ? r : (r&0x3|0x8);
+    return v.toString(16);
+  });
```

Comment:
no like I said this was some hack-a-day stuff. I just pulled a uuid generator off stack overflow.